internal/cli/events/organization/list.go

```diff
@@ -30,7 +30,6 @@ type ListOpts struct {
 	cli.ListOpts
 	cli.OutputOpts
 	orgID     string
-	projectID string
```

Comment:
can't select the line with the problem but `store.EventLister` has probably both methods for projects and orgs, for the new commands you should have a new interface only with the methods you need